Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedBackingStore.cpp

```diff
@@ -36,10 +37,18 @@ void CoordinatedBackingStoreTile::addUpdate(Update&& update)
 void CoordinatedBackingStoreTile::swapBuffers(TextureMapper& textureMapper)
 {
     auto updates = WTFMove(m_updates);
-    for (auto& update : updates) {
+    auto updatesCount = updates.size();
+    if (!updatesCount)
+        return;
+
+    WTFBeginSignpost(this, CoordinatedSwapBuffers, "%lu updates", updatesCount);
```

Comment:
Would it make sense to also produce a signpost when `updatesCount` is zero, too? Feel free to add it before landing if that's something we would like to have.

Reply:
I found the absence in the timeline sufficient for understanding. Will think about it again, if it turns out to be a problem we can fix it later.